bears/haskell/HaskellLintBear.py

```diff
@@ -19,7 +23,12 @@ class HaskellLintBear:
     """
 
     LANGUAGES = {'Haskell'}
-    REQUIREMENTS = {DistributionRequirement(apt_get='hlint')}
+    REQUIREMENTS = {AnyOneOfRequirements(
```

Comment:
put `Any...` on the next line indented, and keep indenting for each new level, and always add a trailing comma.